core/lib/env/env.mjs

```diff
@@ -52,6 +52,25 @@ export const env = createEnv({
 		DATACITE_REPOSITORY_ID: z.string().optional(),
 		DATACITE_PASSWORD: z.string().optional(),
 		SENTRY_AUTH_TOKEN: z.string().optional(),
+		DISABLED_ACTIONS: z
+			.string()
+			.transform((value) => value.split(","))
+			.pipe(
+				z
+					// TODO: Figure out how to get `import {actionSchema} from "db/public"` working
+					.enum([
+						"log",
+						"pdf",
+						"email",
+						"pushToV6",
+						"http",
+						"move",
+						"googleDriveImport",
+						"datacite",
```

Comment:
I was unable to import anything from `db/public` in this `.mjs` module. @tefkah do you have any insights here? It'd be nice to not have to duplicate this enum.

Reply:
I will fix this in a follow-up branch named `em/flags`.